### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by marcoow**
🆙

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,41 +20,44 @@
...
+    "ember-cli-fastboot": "^1.0.0-rc.5",
+    "ember-cli-htmlbars": "^2.0.2",
+    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
+    "ember-cli-inject-live-reload": "^1.6.1",
     "ember-cli-mocha": "^0.14.4",
-    "ember-cli-shims": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-cli-shims": "^1.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^1.2.0",
```
Comment (Turbo87) [inline package.json]: we don't need `sri` and `uglify`

File: package.json [hunk 2]
```diff
@@ -20,41 +20,44 @@
...
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-export-application-global": "^2.0.0",
```
Comment (Turbo87) [inline package.json]: don't need this one either

File: package.json (hunk 1 above)
Comment (marcoow) [inline package.json]: This is added by `ember init` - I'd add that so it doesn't come up again and again on every ember-cli update?

File: package.json (hunk 2 above)
Comment (marcoow) [inline package.json]: see above

File: package.json (hunk 1 above)
Comment (Turbo87) [inline package.json]: but we don't actually **need** them...!

Comment (marcoow) [inline package.json]: removed

File: package.json (hunk 2 above)
Comment (marcoow) [inline package.json]: removed

**Comment by Turbo87** [comment]
@marcoow needs a rebase

**Comment by marcoow** [comment]
@Turbo87 done

**Pull request merged**